apps/coscrad-frontend/src/emotion-components/colors.ts

```diff
@@ -0,0 +1,7 @@
+export const primaryColor = {
+    100: '#F53E20',
```

Comment:
Are the numbers, e.g. `100`, to line up different options other groups can choose from pallets?

I'm not sure I like this approach. We want a pattern where we inject these colour codes form a config at run-time. We want groups modifying their config, not the code.